packages/slice-machine/src/features/onboarding/OnboardingGuide/OnboardingGuide.module.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  overflow: hidden;
+  position: relative;
+  opacity: 1;
+  visibility: visible;
+  transition: opacity 500ms linear;
+}
+
+.confettiCannon {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.invisible {
+  opacity: 0;
+  visibility: hidden;
+  transition-delay: 2000ms;
```

Comment:
confetti duration is 3s and this delay is 2s, not sure if they must go hand in hand

Reply:
2s for the fade out to start looked like the best timing (as seen in the video), even if the confetti is still being rendered. If I increase this, the fadeout happens too late.

Reply:
Ah, i see, do you mind leaving a comment there in the CSS about this? I think it's important for the reader to understand that the value is not random.

Reply:
Yeah for sure, done!